src/ui.rs

```diff
@@ -498,7 +500,7 @@ impl<'a> UiFrame<'a> {
             .movable(true)
             .resizable(false)
             .collapsible(false)
-            .always_auto_resize(true)
+            .always_auto_resize(false)
```

Comment:
This makes the about window fixed height and therefore it doesn't extend the window boundaries (so easily).